Formula/yj.rb

```diff
@@ -0,0 +1,19 @@
+class Yj < Formula
+  desc "CLI to convert between YAML, TOML, JSON, and HCL"
```

Comment:
```suggestion
  desc "CLI to convert between YAML, TOML, JSON and HCL"
```
See our [Prose Style Guidelines](https://docs.brew.sh/Prose-Style-Guidelines#typographical-conventions).